content/docs/command-reference/remote/modify.md

```diff
@@ -246,17 +246,18 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify myremote listobjects true
   ```
 
-- `sse` - server-side encryption algorithm to use (e.g. AES256, aws:kms). By
+- `sse` - server-side encryption algorithm to use (e.g. `AES256`, `aws:kms`). By
   default, no encryption is used.
```

Comment:
What other options are there?

Does `AES256` require any other params to be set?

@dberenbaum @efiop thanks

Reply:
> What other options are there?

Those are the only valid values (see https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html).

> Does `AES256` require any other params to be set?

No, AWS automatically manages the keys in that case.